src/pages/policy/output/ImpactTypes.jsx

```diff
@@ -13,26 +13,25 @@ import povertyImpactByGender from "./poverty/PovertyImpactByGender";
 import povertyImpactByRace from "./poverty/PovertyImpactByRace";
 import relativeImpactByDecile from "./decile/RelativeImpactByDecile";
 import relativeImpactByWealthDecile from "./decile/RelativeImpactByWealthDecile";
-import laborSupplyImpact from "./labourSupply/LaborSupplyResponse";
+import LaborSupplyResponseAbsolute from "./labourSupply/LaborSupplyResponseAbsolute";
+import LaborSupplyResponseRelative from "./labourSupply/LaborSupplyResponseRelative";
 import lsrHoursImpact from "./labourSupply/LaborSupplyHoursImpact";
 import {
   LabourSupplyDecileAbsoluteImpactIncome,
   LabourSupplyDecileAbsoluteImpactSubstitution,
   LabourSupplyDecileAbsoluteImpactTotal,
 } from "./labourSupply/LabourSupplyDecileAbsoluteImpacts";
-/*
 import {
   LabourSupplyDecileRelativeImpactIncome,
   LabourSupplyDecileRelativeImpactSubstitution,
   LabourSupplyDecileRelativeImpactTotal,
 } from "./labourSupply/LabourSupplyDecileRelativeImpacts";
-*/
 import Analysis from "./Analysis";
 
 const map = {
   "budgetaryImpact.overall": budgetaryImpact,
   "budgetaryImpact.byProgram": detailedBudgetaryImpact,
-  "laborSupplyImpact.earnings.overall": laborSupplyImpact,
+  // "laborSupplyImpact.earnings.overall": laborSupplyImpact,
```

Comment:
are we waiting on an api change to uncomment this?

Reply:
Ah, no, that's just garbage, thanks for finding